examples/Forms/MultiLineTextInput.md

```diff
@@ -0,0 +1,159 @@
+Basic multi-line text box
+
+```jsx
+import { MultiLineTextInput } from 'mark-one';
+import { useState } from 'react';
+
+const [value, setValue] = useState('CS50 is an on-campus and online introductory course on computer science taught at Harvard University and Yale University.');
+
+<MultiLineTextInput
+  value={value}
+  id="course-notes"
+  label="Course notes"
+  onChange={(event) => {
+    setValue(event.target.value);
+  }}
+/>
+```
+
+Multi-line text box with an error
+```jsx
+import { MultiLineTextInput } from 'mark-one';
+import { useState } from 'react';
+
+const [value, setValue] = useState('');
+
+<MultiLineTextInput
+  value={value}
+  id="course-notes"
+  label="Course notes"
+  errorMessage="Please provide a value for this field"
+  onChange={(event) => {
+    setValue(event.target.value);
+  }}
+/>
+```
+
+Multi-line text box with a hidden error
+```jsx
+import { MultiLineTextInput } from 'mark-one';
+import { useState } from 'react';
+
+const [value, setValue] = useState('');
+
+<MultiLineTextInput
+  value={value}
+  id="course-notes"
+  label="Course notes"
+  errorMessage="Please provide a value for this field"
+  hideError
+  onChange={(event) => {
+    setValue(event.target.value);
+  }}
+/>
+```
+
+
+Multi-line text box with a hidden label
+```jsx
+import { MultiLineTextInput } from 'mark-one';
+import { useState } from 'react';
+
+const [value, setValue] = useState('');
+
+<MultiLineTextInput
+  value={value}
+  id="course-notes"
+  label="Course notes"
+  isLabelVisible={false}
+  onChange={(event) => {
+    setValue(event.target.value);
+  }}
+/>
+```
+
+Multi-line text box with a hidden label
```

Comment:
Looks like there is a duplicated example of a hidden label `MultiLineTextInput`.

Reply:
Whops! Copy/paste strikes again!

Fixed in 76561f8